source/server/spongineer/properties.rst

```diff
@@ -0,0 +1,74 @@
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Adds a StackTrace to each ``StackFrame`` for when either the Sponge             |
+| **sponge.debugcauseframes**        | implementation or a plugin potentially corrupts the ``CauseStackManager`` 's    |
+|                                    | ``StackFrame`` s, whether it's a frame being popped out of order, or popping    |
+| (boolean, default: false)          | an already popped frame. This is useful to find out where the frame was         | 
+|                                    | created.                                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Similar to the ``StackFrame`` exception creation, this tracks the stack trace   |
+|                                    | for each ``BlockTransaction`` that is captured for a block being changed, a     | 
+| **sponge.debugProxyChanges**       | neighbor notification request, or a ``TileEntity`` change. Useful to find out   |
+|                                    | where each transaction is created, potentially adding more information if there |
+| (boolean, default: false)          | is an exception thrown during the processing of said transactions. Should       |
+|                                    | always be included for logs when reporting issues involving these sort of       |
+|                                    | transactions.                                                                   |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.filter.debug**            | Writes out classes that are generated by the filter generator to the            |
+|                                    | ``.sponge.debug.out`` folder, named after the class they are altering.          |
+| (boolean, default: false)          |                                                                                 |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.print_all_catalog_types** | Prints out all registered ``CatalogType`` s and their ``String`` id's at the    | 
+|                                    | end of startup, useful for finding out what id's are registered from vanilla,   |
+| (boolean, default: false)          | mods, and plugins for all created ``CatalogType`` s.                            |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Prints out a very spam filled log for every block captured that is processed,   |
```

Comment:
Do we have better terminology than `spam-filled`? e.g. cluttered, busy, crowded, content-rich, verbose, ...

Reply:
Changed `spam filled` to `verbose log entry`.